index.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
         "dollar-sign"
     ],
     "rules": {
+        // Используем 'use strict' в функциях и модулях
+        "strict": ["error", "safe"],
```

Comment:
> "safe" (default) corresponds either of the following options:
> 
> * "global" if ESLint considers a file to be a CommonJS module
> * "function" otherwise